runtime/j9vm/jvm.c

```diff
@@ -2233,7 +2233,9 @@ JNI_GetCreatedJavaVMs(JavaVM **vmBuf, jsize bufLen, jsize *nVMs)
  */
 
 jint JNICALL JNI_GetDefaultJavaVMInitArgs(void *vm_args) {
-	UDATA requestedVersion = (UDATA)((JDK1_1InitArgs *)vm_args)->version;
+	JDK1_1InitArgs *init_args = (JDK1_1InitArgs *)vm_args;
+	UDATA requestedVersion = (UDATA)init_args->version;
+	init_args->javaStackSize = J9_OS_STACK_SIZE;
```

Comment:
This is only reasonable if `requestedVersion == JNI_VERSION_1_1`. For other versions the type is `JavaVMInitArgs` which isn't large enough for the offset of `javaStackSize`. Please add a case to the `switch` statement below.

Reply:
Changed so setting `javaStackSize` only occurs when version is `JNI_VERSION_1_1`